manifests/server/puppetserver.pp

```diff
@@ -232,4 +248,16 @@
     ensure  => $product_conf_ensure,
     content => template('puppet/server/puppetserver/conf.d/product.conf.erb'),
   }
+
+  # Do we want to leave this untouched for now?
+  # if versioncmp($server_puppetserver_version, '5.0') >= 0 {
```

Comment:
I guess this can be left out for now... otherwise we would need to provide a lot of configuration possibilities